File: Platron/Pages/Helpers/ShopCartHelper.cs
using System;
using Xunit;
using OpenQA.Selenium;

namespace Platron.Pages.Helpers
{
    public class ShopCartHelper : BaseHelper
    {
        public ShopCartHelper CheckExistRowsInBin(int expectedElement = 0)
        {
            WaiteUntilPageLoad();
            var temp = driverHelper.FindElements(By.XPath(UIShopCartPage.RowsOnBinPage));
            Assert.Equal(expectedElement, temp.Count);
            return this;
        }

        public ShopCartHelper ActionRemoveElementFromBin()
        {
            var element = driverHelper.FindElement(By.XPath(UIShopCartPage.RowsOnBinPage))
                .FindElement(By.XPath(UIShopCartPage.RemoveRowOnBinPage));
            element.Click();
            WaitInvisibilityOfElementLocated(UIShopCartPage.RowsOnBinPage);
            return this;
        }
    }
}

File: Platron/src/Pages/ShopCartHelper.java
import java.util.List;
import org.junit.Assert;
import org.openqa.selenium.By;
import org.openqa.selenium.WebElement;

public class ShopCartHelper extends  BaseHelper {
    ShopCartPage UIShopCartPage = new ShopCartPage();
    public ShopCartHelper()
    {
        super();
    }

    public ShopCartHelper CheckExistRowsInBin(int expectedElement) throws Exception {
        WaiteUntilPageLoad(10);
        List<WebElement> temp = driverHelper.findElements(By.xpath(UIShopCartPage.RowsOnBinPage));
        Assert.assertEquals("Size not correct", expectedElement, temp.size());
        return this;
    }
    public ShopCartHelper ActionRemoveElementFromBin()
    {
        WebElement element = driverHelper.findElement(By.xpath(UIShopCartPage.RowsOnBinPage))
                .findElement(By.xpath(UIShopCartPage.RemoveRowOnBinPage));
        element.click();
        WaitInvisibilityOfElementLocated(UIShopCartPage.RowsOnBinPage);
        return this;
    }

}
